src/main/java/persistence/sql/config/PersistenceConfig.java

```diff
@@ -69,7 +69,7 @@ public EntityPersister entityPersister() throws SQLException {
     }
 
     public PersistenceContext persistenceContext() throws SQLException {
```

Comment:
```suggestion
    public PersistenceContext persistenceContext() {
```
사소하지만, throws 구문은 없어져도 될 것 같네요 👍